net/etesync-server/files/81_setup-etesync-server

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+[ "${PKG_UPGRADE}" = "1" ] && /etc/init.d/etesync-server stop
+
+cd /usr/share/etesync-server || exit 1
+
+python3 manage.pyc migrate --noinput || exit 1
+
+# setup minimal ini for collectstatic:
+mkdir -p /var/etc/etesync-server/ || exit 1
+printf "[global]\nSTATIC_ROOT=/www/etesync/static" >etesync-server.ini || exit 1
+python3 manage.pyc collectstatic --noinput || exit 1
+
+ETESYNC_HAS_USER_PY3CMD="import sqlite3
+c = sqlite3.connect('db.sqlite3').cursor()
+c.execute('select * from auth_user')
+if c.fetchone()==None: print('0')
+else: print('1')"
+
+echo
+
+if [ "$(python3 -c "$ETESYNC_HAS_USER_PY3CMD" || exit 1)" = "0" ]
+then
+    echo "===== First, create a superuser of the Webinterface by ====="
+    [ -t 0 ] && python3 manage.pyc createsuperuser ||
```

Comment:
I'm guessing if the user installs this package in luci, then this will not be called? And the server will still be started by the init script, without a superuser?

Reply:
Hmm, I will think again about it, but I did not see a good solution (cf. [above](https://github.com/openwrt/packages/pull/9865#discussion_r388388744)) ...

I could create a superuser with a default password (or a random one) and print  a message about that if the shell is not interactive (and else still call the interactive command). Would that be better?

For now, there is only a message that you have to invoke `python3 manage.pyc createsuperuser`. The server will be started in any case and you can see the login interface at `$LOCALIP/etesync/admin`, but you cannot login without superuser. 

And there are some more steps if you want to reach it from WAN not just locally: you must setup a server in Nginx and add the corresponding domain/ip as allowed_host to the config. There is a [template](https://github.com/etesync/server/blob/master/templates/success.html) available at `$LOCALIP/etesync` linking to the upstream README. I am thinking about:
* using the allowed_host from Nginx configuration where possible
* extending the installed template with some more information
* maybe even creating a LuCI app

But, I think they are better in further PRs …